### DISCUSSION TITLE
Does CVA lib causes runtime overhead like other css-js/ts libs?

### DISCUSSION THREAD

**Discussion by DeveloperMatheus** [joe-bell/cva#83] [Q&A] [answered]
I'm using cva and i'm loving it! I know this lib can be a really good alternative to avoid css-in-js, but i have this concern related to performance on my projects...

**Comment by joe-bell** [comment, marked as answer]
> **Disclaimer**: Although `cva` is a [**tiny**](https://bundlephobia.com/package/class-variance-authority) library, it's best to use in a SSR/SSG environment – your user probably doesn't need this JavaScript, especially for static components.

https://github.com/joe-bell/cva#getting-started

---

cva is currently [572B minizipped](https://bundlephobia.com/package/class-variance-authority@0.4.0), so really you should be fine – _but_ there's no harm in moving that to a server-side/static function if you're concerned

**Reply by MadadiSaeed** [reply]
Hi. Do you have any example to move cva to server-side in case of remix or next? I appreciate if you can help.

**Reply by joe-bell** [reply]
Sure: you can use React.js Server Components, or you could supply via Next.js' `getStaticProps`, or Remix loaders

**Reply by MadadiSaeed** [reply]
Thanks for the response.
